MusicAppService/services/service.js

```diff
@@ -343,6 +343,70 @@ async function getSongsByPlayListId(playlistId) {
   }
 }
 
+// creates a avatar
+async function createAvatar(_id, avatarUrl) {
+  try {
+    const userData = await Users.findById({ _id });
+    const getavataruser = await Users.findByIdAndUpdate(
+      { _id: _id },
+      {
+        avatarUrl: avatarUrl
+      },
+      {
+        new: true
+      }
+    );
+    return getavataruser;
+  } catch (error) {
+    return error;
+  }
+}
+
+//get avatar
+async function getAvatar(_id) {
+  try {
+    const userData = await Users.findById({ _id });
+    return userData.avatarUrl;
+  } catch (error) {
+    return error;
+  }
+}
+
+//add comment to playlist
+async function addComment(email, comment) {
+  try {
+    const playlist = await Playlist.find({
+      $where: "this.email === '" + email + "'"
+    });
+    // const userData = await Users.getUserPlayList({ _id });
+    const comments = await Users.findByIdAndUpdate(
+      { email: email },
+      {
+        comment: comment
+      },
+      {
+        new: true
+      }
+    );
```

Comment:
comment is in playlist schema, why find is done from users?

And, where is add happening?